index.html

```diff
@@ -44,7 +44,7 @@ <h3>Dorothy</h3>
 							</div>
 						</div>
 						<div class="friend">
-							<h3>Lion</h3>
+							<h3>Cowardly Lion</h3>
 							<div>
 								<img src="images/lion.jpg" alt="Lion">
 							</div>
```

Comment:
Would you want to change the alt to "Cowardly Lion" as well?

Reply:
yes definitely! Thanks

Reply:
Changed the alt. Thanks for the headsup!

Reply:
Changed the alt. Thanks for the headsup!